### PR TITLE
Update mediaelement.js to latest version

### PR DESCRIPTION
**Pull request by Dananji**

### CODE REVIEW

**Comment by Dananji** [comment]
@cjcolvar Thanks for finding the notes :pray: I couldn't find it. I will check on the things you've mentioned and in the my notes from last time.

File: vendor/assets/javascripts/mediaelement/mediaelement-and-player.js [hunk 1]
```diff
@@ -2269,14 +2418,10 @@ Object.assign(_player2.default.prototype, {
 		    chaptersTitle = (0, _general.isString)(t.options.chaptersText) ? t.options.chaptersText : _i18n2.default.t('mejs.captions-chapters'),
 		    total = player.trackFiles === null ? player.tracks.length : player.trackFiles.length;
 
-		try {
```
[the remark this answers is not in the document]
Comment (Dananji) [reply vendor/assets/javascripts/mediaelement/mediaelement-and-player.js:2272]: I tested on browserstack locally for this, and it's working without any issues. But I think we should do thorough testing on each browser.

File: vendor/assets/javascripts/mediaelement/mediaelement-and-player.js [hunk 2]
```diff
@@ -2426,22 +2571,19 @@ Object.assign(_player2.default.prototype, {
 	cleartracks: function cleartracks(player) {
 		if (player) {
 			if (player.captions) {
-				delete player.captions;
```
[the remark this answers is not in the document]
Comment (Dananji) [reply vendor/assets/javascripts/mediaelement/mediaelement-and-player.js]: This was done as part of the work for preserving fullscreen mode when playing through sections. I reverted this to the customization's version we had. Thanks for catching this.

**Review by cjcolvar — APPROVED** [review]
Look's good.  Testing should hopefully find anything that may have gotten missed.

**Pull request merged**